src/components/EventPreview/EventPreview.tsx

```diff
@@ -35,7 +35,7 @@ export interface EventPreviewProps {
    */
   strings?: PureObject;
   /**
-   * List of event properties to be hidden. Possible values: type, subtype, description, startTime, endTime, metadata
+   * List of event properties to be hidden. Possible values:
```

Comment:
List of event properties to be hidden. Possible values: ->
List of event properties to be hidden